packages/patternfly-4/react-core/README.md

```diff
@@ -112,3 +112,52 @@ yarn test packages/patternfly-4/react-core
 [docs]: https://patternfly-react.surge.sh/patternfly-4
 
 
+## Tree Shaking
+
+Ensure optimization.sideEffects is set to true within your Webpack config:
+```JS
+optimization: {
+  sideEffects: true
+}
+```
+
+Use ESM module imports to enable tree shaking with no additional setup required.
+```JS
+import { TimesIcon } from '@patternfly/react-icons';
+```
+
+To enable tree shaking with named imports for CJS modules, utilize [babel-plugin-transform-imports](https://www.npmjs.com/package/babel-plugin-transform-imports) and update a babel.config.js file to utilize the plugin:
+```JS
+require.extensions['.css'] = () => undefined;
+const components = require('@patternfly/react-core/dist/js/components');
```

Comment:
our package.json files have `main` and `module` entries, when doing require it should find the cjs entry point as specified in `main` and when doing imports it should find the esm entry point in `module`. So I don't think you need the whole path?

Reply:
anyways, might be different for babel js resolution. looks good :)